docs/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: VMware VRA
+meta_desc: Provides an overview of the VMware VRA Provider for Pulumi.
+layout: overview
+---
+
+The VMware VRA provider for Pulumi can be used to provision a VMware VRA based installation.
+
+## Example
+
+{{< chooser language "typescript,python,go,csharp" >}}
+{{% choosable language typescript %}}
+
+```typescript
+
+// I'm not sure.
+
+```
+
+{{% /choosable %}}
+{{% choosable language python %}}
+
+```python
+
+// I'm not sure.
```

Comment:
If you start from the `yaml` example above, you can use `pulumi convert` to generate for languages you are less familiar with yourself.

https://www.pulumi.com/blog/pulumi-yaml-ga/#full-support-for-pulumi-convert